src/components/login/loginForm/loginForm.scss

```diff
@@ -0,0 +1,8 @@
+.loginForm {
+  width: 242px;
+  margin: 0 auto;
+}
+
+.loginForm div {
+  margin: 8px auto;
+}
```

Comment:
using scss you can write it this way:

.loginForm {
  width: 242px;
  margin: 0 auto;
  
  div {
    margin: 8px auto;
  }
}

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/components/login/loginForm/loginForm.scss#L1-L8